.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
       - name: 💄 Ensure Wasm up to date
         if: contains(matrix.os, 'ubuntu')
-        run: deno task build --check
+        run: RUSTFLAGS= deno task build --check
```

Comment:
I removed `RUSTFLAGS: -D warnings`, because the wasm build is complaining about useless stuff like public methods not being used. I don't think it's worth adding a bunch of `#[allow(unused)]` to these just for Wasm considering the compiler should optimize this away.